AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5056,6 +5056,9 @@ vi.nl#%#AG_onLoad(function() { if(window.location.href.indexOf("cookies") != -1)
 volkskrant.nl#%#AG_onLoad(function() { setTimeout(function() {if(document.getElementsByClassName("modal-overlay")[0].isvisible = true){document.getElementsByClassName("button button--accept")[0].click();}}, 300); });
 ziekenhuis.nl#%#AG_onLoad(function() { setTimeout(function() {var check = document.getElementsByClassName('cookiewall')[0]; if (check){document.cookie = "acceptcookies=true"; location.reload();}; }, 300); });
 !
+stockmann.com#$#body { position: static; }
+stockmann.com##div.cookie-dialog-screen.open
+!
```

Comment:
for such cases rules should be type alike. I mean that both should be `#$#` and have `important` modifier. so eventually they should be:
```
stockmann.com#$#body { position: static!important; }
stockmann.com#$#.cookie-dialog-screen { display: none!important; }
```

could you please fix that?

Reply:
Oh, sure 👍